### PR TITLE
Generate docs & prepare for release

### PR DESCRIPTION
**Pull request by jethrogb**

### CODE REVIEW

File: doc/generate-api-docs.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    | egrep '^'$(echo $LIBS|sed 's/ /|/g')'$'
+)
+
+for LIB in $LIBS_SORTED; do
+    cd $LIB
+    ARGS=""
+    if FEATURES="$(cargo read-manifest|jq -r '.metadata.docs.rs.features | join(",")' 2> /dev/null)"; then
+        ARGS="--features $FEATURES"
+    fi
+    cargo doc --lib $ARGS
```
Comment (akash-fortanix) [inline doc/generate-api-docs.sh]: cargo doc --no-deps --lib $ARGS

File: doc/generate-api-docs.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+# Generate docs for core/alloc/std (x86_64-fortanix-unknown-sgx target)
+if [ -v RUST_DIR ]; then
+    PATCH="$(readlink -f "$(dirname "$0")/rust-std-edp-doc.patch")"
+    pushd $RUST_DIR
+    COMMIT=$(rustc --version -v|sed -n 's/^commit-hash: //p')
+    git checkout $COMMIT
+    git reset --hard $COMMIT
+    git submodule update --init --recursive
+    git apply < $PATCH
```
Comment (akash-fortanix) [inline doc/generate-api-docs.sh]: A valid patch doesn't exist at this point

Comment (jethrogb) [reply doc/generate-api-docs.sh]: Works for me? Is your nightly up to date?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,30 @@
...
 
-The Fortanix Rust EDP is the preferred way to write Intel SGX enclaves from scratch.
+The [Fortanix Rust EDP](https://edp.fortanix.com/) is the preferred way to write Intel SGX enclaves from scratch.
 
-## [Get started now.](https://github.com/fortanix/rust-sgx/issues/49)
+## [Get started now.](https://edp.fortanix.com/docs/installation/guide/)
+
+```sh
+# Install EDP components
+rustup add target x86_64-fortanix-unknown-sgx
```
Comment (akash-fortanix) [inline README.md]: Nightly?

File: dcap-provider/Cargo.toml [hunk 4]
```diff
@@ -23,7 +23,8 @@ DCAP: DataCenter Attestation Primitives
 QL: Quoting Library
 """
 repository = "https://github.com/fortanix/rust-sgx"
-documentation = "https://docs.rs/dcap-provider/"
+documentation = "https://edp.fortanix.com/docs/"
```
Comment (akash-fortanix) [inline dcap-provider/Cargo.toml]: documentation = "https://edp.fortanix.com/docs/api/dcap-provider" ?

File: aesm-client/src/lib.rs [hunk 5]
```diff
@@ -8,6 +8,10 @@
 //! * `sgxs`. Enable the `sgxs` feature to get an implemention of
 //!   `EinittokenProvider` that uses AESM.
 
+#![doc(html_logo_url = "https://edp.fortanix.com/api/edp-logo.svg",
+       html_favicon_url = "https://edp.fortanix.com/favicon.ico",
+       html_root_url = "https://edp.fortanix.com/api/")]
```
Comment (akash-fortanix) [inline aesm-client/src/lib.rs]: Are you sure about the html_root_url? I didn't find any issue with navigation but was hoping to see https://edp.fortanix.com/docs/api/

File: dcap-provider/Cargo.toml (hunk 4 above)
Comment (jethrogb) [reply dcap-provider/Cargo.toml]: No, this crate doesn't have any useful documentation. It doesn't have any documentation right now but it's not going to live there

File: aesm-client/src/lib.rs (hunk 5 above)
Comment (jethrogb) [reply aesm-client/src/lib.rs]: Will fix

File: README.md (hunk 3 above)
Comment (jethrogb) [reply README.md]: Ok

**Pull request merged**